CMakeLists.txt

```diff
@@ -231,7 +231,7 @@ if (DYAD_LOGGER STREQUAL "CPP_LOGGER")
                        ${CPP_LOGGER_PATH} ${cpp-logger_PATH}
                        $ENV{CPP_LOGGER_DIR} $ENV{cpp-logger_DIR}
                        $ENV{CPP_LOGGER_PATH} $ENV{cpp-logger_PATH})
-    if (${CPP_LOGGER_FOUND})
+    if (${cpp-logger_FOUND})
```

Comment:
You can look for lib version 1.0.0 in the find package. Also, switch CI to install 0.0.4. Thoughts?

Reply:
Not sure I really understand what you mean here. We can discuss briefly in our meeting later today.

Reply:
I have a PR for spack to include latest cpp logger. https://github.com/spack/spack/pull/45033 Once this is approved we can do this change.

Reply:
Ah, I see now. Yeah, we should hold off until that gets merged.